extensions/Worker.Extensions.CosmosDB/src/Utilities.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    internal class Utilities
```

Comment:
I prefer Utilities since there could be more than one utility method. This should be covered via an existing test but I can write some test methods for this function!